mlflow/alembic/versions/451aebb31d03_add_metric_step.py

```diff
@@ -0,0 +1,35 @@
+"""add metric step
+
+Revision ID: 451aebb31d03
+Revises: 
+Create Date: 2019-04-22 15:29:24.921354
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '451aebb31d03'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('metrics', sa.Column('step', sa.BigInteger(), nullable=False, server_default='0'))
+    # Use batch mode so that we can run "ALTER TABLE" statements against SQLite
+    # databases (see more info at https://alembic.sqlalchemy.org/en/latest/
```

Comment:
Do we have a test which verifies that the final state of the SQL schema is some expected value? This helps ensure that we have the full schema documented somewhere (initial + updates); otherwise, it's kind of hard to reconstruct.